### ISSUE TITLE
ffdhe groups not supported 

### ISSUE THREAD

**Issue by tomato42** [openssl/openssl#6519]
## Issue:
when starting openssl with `-groups` option that includes ffdhe groups, the command aborts:
```
openssl s_server -psk aa -nocert -psk_identity test -www\
 -allow_no_dhe_kex -groups P-256:X25519:ffdhe2048
```
output:
```
Error with command: "-groups P-256:X25519:ffdhe2048"
```

But when the `:ffdhe2048` is omitted, the server starts and supports both groups.

## Context:
Support for ffdhe is especially interesting to people that want to harden their communications against quantum computers before wide availability of PQE.

**Comment by mattcaswell** [comment]
Yes, it's a known missing feature.

**Comment by tomato42** [comment]
This was resolved with #8178
